airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_39_1_001__CreateResets.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.jooq.impl.DSL.currentOffsetDateTime;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO: update migration description in the class name
+public class V0_39_1_001__CreateResets extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_39_1_001__CreateResets.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    // Warning: please do not use any jOOQ generated code to write a migration.
+    // As database schema changes, the generated jOOQ code can be deprecated. So
+    // old migration may not compile if there is any generated code.
+    final DSLContext ctx = DSL.using(context.getConnection());
+    createResetsTable(ctx);
+  }
+
+  private static void createResetsTable(final DSLContext ctx) {
+    final Field<UUID> id = DSL.field("id", SQLDataType.UUID.nullable(false));
+    final Field<UUID> connectionId = DSL.field("connection_id", SQLDataType.UUID.nullable(false));
+    final Field<String> streamName = DSL.field("stream_name", SQLDataType.VARCHAR(256).nullable(false));
```

Comment:
Do we have limitation on the stream name size? I think it should be a field without limitation.

Reply:
the spec has it as a varchar but doesn't have a limit listed. i could make it a text field instead - what do you think?

Reply:
I think we should avoid having a limit because none is defined right now. So going for text type sounds right.